ts/navigation/MainNavigator.tsx

```diff
@@ -1,12 +1,6 @@
-/**
```

Comment:
why you removed the comment?

Reply:
Sorry, I didn't even notice it. Most likely I removed it while importing a library

Reply:
Still removed